x-pack/plugins/security_solution/server/lib/timeline/saved_object/notes/saved_object.ts

```diff
@@ -132,24 +130,7 @@ const createNote = async ({
   const savedObjectsClient = (await request.context.core).savedObjects.client;
   const userInfo = request.user;
 
-  const shallowCopyOfNote = { ...note };
-  let timelineVersion: string | undefined;
-
-  if (note.timelineId == null) {
```

Comment:
This was the piece of code that would create a timeline when no `timelineId` was provided. Which is not needed anymore.